src/main/resources/db/migration/V1_11__ATTENDANCE.sql

```diff
@@ -0,0 +1,26 @@
+ALTER TABLE `ATTENDANCE`
+    ADD COLUMN `CREATED_DATE` datetime,
+    ADD COLUMN `UPDATED_DATE` datetime;
+
+UPDATE `ATTENDANCE`
+  SET CREATED_DATE = '2018-01-09', UPDATED_DATE = '2018-01-09'
```

Comment:
hm, we need more users in one lesson, so please add more entries for same dates.